_posts/2018-01-13-DAY-3-solution-in-go.md

```diff
@@ -0,0 +1,73 @@
+---
+published: true
```

Comment:
Add title, author, layout and date

[Read this](https://jekyllrb.com/docs/posts/)

Reply:
@mubaris what should be the title? and what do you mean by layout?

Reply:
Put title something meaningful, not "Day 3 Solution". Create a title from the question

and layout is post

```
layout: post
```